src/js/menu/menu-item.js

```diff
@@ -51,15 +63,22 @@ class MenuItem extends Button {
    * @method selected
    */
   selected(selected) {
-    if (selected) {
-      this.addClass('vjs-selected');
-      this.el_.setAttribute('aria-selected',true);
-    } else {
-      this.removeClass('vjs-selected');
-      this.el_.setAttribute('aria-selected',false);
+    if (this.selectable) {
+      if (selected) {
+        this.addClass('vjs-selected');
+        this.el_.setAttribute('aria-checked',true);
+        // aria-checked isn't fully supported by browsers/screen readers,
+        // so indicate selected state to screen reader in the control text.
+        this.controlText(', selected');
+      } else {
+        this.removeClass('vjs-selected');
+        this.el_.setAttribute('aria-checked',false);
+        // Indicate un-selected state to screen reader
+        // Note that a space clears out the selected state text
+        this.controlText(' ');
```

Comment:
We could rename `controlText` as `accessibilityText`, since it's the text equivalent for the visual of the button. For the play/pause button, the visual is from the videojs font set and looks like ▶(which is not announced by screen readers), while the `controlText` is `Play` or `Pause`.

For `menu-item`s, the visual is 'English captions' (which is handled correctly by screen readers), so `controlText` is not needed or used. I added the ', selected' or ' ' as essentially an alternate use of the `controlText` field.